File: ej6-numeros_literales_sol1.rb
# Crear una lista para "emparejar" los números con sus literales
# 1 <---> uno
# 2 <---> dos
# 3 <---> tres
# ...

num_array = Array(0..10)

literals_array = ['cero','uno', 'dos','tres', 'cuatro', 'cinco', 'seis', 'siete', 'ocho', 'nueve', 'diez']

puts "ingresa un valor"
num = gets.to_i

if num_array.include?(num)
  puts literals_array[num]
else
  puts "fuera de rango"
end

File: ej6-numeros_literales_sol1.js
let literals_array = ['cero','uno', 'dos','tres', 'cuatro', 'cinco', 'seis', 'siete', 'ocho', 'nueve', 'diez']

let n = parseInt(prompt("Ingrese el valor de n"))

// prompt entrega texto, JS lo interpreta abajo y automaticamente lo transforma en número, pero internametne lo que hace es la función parseInt.
// Si queremos asegurarnos de que el valor se transforme a integer, agregamos el parseInt

if (n>0 && n<11){
  console.log(literals_array[n-1])
}
else {
  console.log("???")
}